### PR TITLE
feat(github plugin): resolve github commits and pull requests

### PR DESCRIPTION
**Pull request by warren830**
# Summary
1. Extracted func CollectPullRequestCommits from CollectChildrenOnPullRequests as a standalone task
2. Resolved the relationship between pr and commit and stored as a new table pr_commit

### Screenshots
image

### Does this close any open issues?
Will close issue #950

### CODE REVIEW

**Comment by klesh** [comment]
pr/commits relationship is reprsented in comma separated format? it doesn't seem right.

**Comment by warren830** [comment]
> pr/commits relationship is reprsented in comma separated format? it doesn't seem right.

Fixed in the new pr

File: plugins/github/tasks/github_pull_request_commit_convertor.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+func PrCommitConvertor() (err error) {
+	githubPullRequestCommit := &models.GithubPullRequestCommit{}
+
+	cursor, err := lakeModels.Db.Model(&githubPullRequestCommit).Rows()
+	if err != nil {
+		return err
+	}
+	defer cursor.Close()
+	// iterate all rows
```
Comment (klesh) [inline plugins/github/tasks/github_pull_request_commit_convertor.go:18]: i think we should delete pr/commits relationship before insertion considering pr might be force pushed.

Comment (warren830) [reply plugins/github/tasks/github_pull_request_commit_convertor.go:18]: I deleted the old relationship when collecting, is that right?

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**